plugins/registry.js

```diff
@@ -517,10 +517,11 @@ export default class PluginRegistry {
     // Accepts the following:
     // - key - A key specified in the settings_schema.settings block of the plugin's manifest.
     // - component - A react component to render in place of the default handling.
-    // - title - Optional title that, if available, will be displayed on the left column
-    // on the settings page and the registered component will be rendered on space available in the right column.
-    registerAdminConsoleCustomSetting(key, component, title) {
-        store.dispatch(registerAdminConsoleCustomSetting(this.id, key, component, title));
+    // - showTitle - Optional boolean that if true the display_name of the setting will be rendered
```

Comment:
See comments on demo plugin, but I'm wondering if we should accepts an options param of some kind to avoid the blind `true` in the call.

Reply:
Wrapped `showTitle` in an `options` parameter to avoid the blind `true`